modules/Sources/Generated/Resources/Localizable.strings

```diff
@@ -123,7 +123,7 @@
 "balances.fee" = "(Fee %@)";
 "balances.synced" = "Synced";
 "balances.syncing" = "Syncing";
-"balances.hintBox.message" = "Zashi uses the latest network upgrade and does does not support sending transparent (unshielded) ZEC. Converting your funds will move them to your available balance so you can send or spend them.";
+"balances.hintBox.message" = "Zashi uses the latest network upgrade and does not support sending transparent (unshielded) ZEC. Use the Shield and Consolidate button to move your funds from transparent to shielded. This will add to your available balance and make your ZEC spendable.";
```

Comment:
```suggestion
"balances.hintBox.message" = "In order to better preserve your privacy, Zashi does not support spending transparent (unshielded) ZEC. Use the Shield and Consolidate button to shield your transparent funds. This will move the transparent value to your available balance and make it spendable.";
```